src/services/tests/parser.service.test.ts

```diff
@@ -0,0 +1,256 @@
+import { AsyncAPIDocument } from '@asyncapi/parser';
+import { Request } from 'express';
+
+import { ParserService } from '../parser.service';
+import { ProblemException } from '../../exceptions/problem.exception';
+
+describe('ParserService', () => {
+  const parserService = new ParserService();
+  const req = {
+    header() { return ''; },
+  } as unknown as Request;
```

Comment:
do we need that `as unknown` or could we simply have `as Request`?

Reply:
If I specify `as Request`, TS will assume that the value does not match the given type (because I did not specify other required fields) - that's why there is this trick with unknown and then Request :) This is a simple mock to test.

Reply:
ahh cool, I learned something new 😄